pkg/controller/clusterprovision/clusterprovision_controller.go

```diff
@@ -290,6 +300,7 @@ func (r *ReconcileClusterProvision) reconcileRunningJob(instance *hivev1.Cluster
 
 func (r *ReconcileClusterProvision) reconcileSuccessfulJob(instance *hivev1.ClusterProvision, job *batchv1.Job, pLog log.FieldLogger) (reconcile.Result, error) {
 	pLog.Info("install job succeeded")
+	metricClusterProvisionsTotal.WithLabelValues(hivemetrics.GetClusterDeploymentType(instance), resultSuccess)
```

Comment:
I assume that you intended to add a `.Inc()` to the end so that the metric is incremented.

Reply:
Ughhhh yes.